### ISSUE TITLE
Great Hack 2.9

### ISSUE THREAD

**Issue by Wortlaut** [ddvk/remarkable-hacks#348]
Thanks a lot. It works perfect and make life much easyer

**Comment by martin-braun** [comment]
Show him some love and sponsor him a few bucks for his service. Just a thought. :)

**Comment by Wortlaut** [comment]
I agree with your thougt . And for this reason, I pay - monthly. nearly since one year ;-) And the hack is is worth every penny

**Comment by dansbandit** [comment]
Thanks for the 2.9 and 2.10 update, @ddvk!

I started to sponsor you as well. Your hacks make it easier to use my remarkable.

**Comment by folofjc** [comment]
Agree, sponsoring is definitely worth it! And thanks for the image describing the different motions, there were a couple I wasn't aware of!
